tools/untils.py

```diff
@@ -0,0 +1,13 @@
+"""This file defined all the objects will been used in ouiija"""
```

Comment:
2 things:
1) comment should say SETA instead of ouija
2) lets rename this to utils.py instead of untils.py